src/components/autofill/autofill.ts

```diff
@@ -100,15 +99,21 @@ export class Autofill extends Component<Props, SpreadsheetChildEnv> {
       this.env.model.dispatch("AUTOFILL");
     };
 
-    const onMouseMove = (ev: MouseEvent) => {
-      const position = gridOverlayPosition();
+    const onMouseMove = (col: HeaderIndex, row: HeaderIndex, ev: MouseEvent) => {
       const { offsetY, offsetX } = this.env.model.getters.getActiveSheetScrollInfo();
-      this.state.position = {
-        left: ev.clientX - start.left + offsetX,
-        top: ev.clientY - start.top + offsetY,
-      };
-      const col = this.env.model.getters.getColIndex(ev.clientX - position.left);
-      const row = this.env.model.getters.getRowIndex(ev.clientY - position.top);
+      if (this.props.position.left > 0 && this.props.position.top > 0) {
+        // autofill selected region still visible
+        this.state.position = {
+          left: ev.clientX - start.left + offsetX,
+          top: ev.clientY - start.top + offsetY,
+        };
+      } else {
+        this.state.position = {
+          left: ev.clientX,
+          // TODO: 60 is hardcoded. We need more advanced method to determine the value, considering the autofill CSS position will become negative numbers when the autofill selected region is invisible, and it has impact on tooltip & handler whose CSS position is absolute which relates to the position of autofill itself.
+          top: ev.clientY - 60,
+        };
```

Comment:
I tried many calculations using `props.position` (the position value passed from `grid`), etc., and this is the one that works best under different scenes when the autofill selected region is invisible. 
The only point is `60` is hardcoded and I don't really know where it comes from... I got this value based on screenshot measurement and some trial-and-error 😅 Any idea is welcome

Reply:
Solved, `60` is come from `top` of the grid canvas.